cert/aziot-certd/openapi/2020-09-01.yaml

```diff
@@ -0,0 +1,128 @@
+# Ref: https://spec.openapis.org/oas/v3.0.3
+
+openapi: '3.0.3'
+
+
+info:
+  title: 'aziot-certd API specification'
+
+  version: '2020-09-01'
+
+  description: |
+    This is the specification of the HTTP API of the aziot-certd service.
+
+  license:
+    name: 'Apache 2.0'
```

Comment:
License should be MIT (same for keyd file below)

Reply:
Fixed.

Reply:
Links (the line after license name) is still pointing to Apache license.

Reply:
Ah shit, I thought I was pushing to my private branch.

Reply:
Anyway, also fixed now.